copis/gui/main_frame.py

```diff
@@ -15,11 +15,16 @@
 
 """MainWindow class."""
 
-from ctypes import *
-from pathlib import Path
-
 import wx
 import wx.lib.agw.aui as aui
+
+from wx.lib.agw.aui.aui_constants import (AUI_NB_BOTTOM,
+    AUI_BUTTON_STATE_HIDDEN, AUI_NB_CLOSE_ON_TAB_LEFT, AUI_BUTTON_STATE_HOVER,
+    AUI_BUTTON_STATE_PRESSED)
+from wx.lib.agw.aui.aui_utilities import (ChopText, GetBaseColour,
+    IndentPressedBitmap, StepColour, TakeScreenShot)
+
+from copis.store import load as store_load, save as store_save
```

Comment:
Why rename when you can just `import copis.store as store` and do `store.load()` and `store.save()`?